### DISCUSSION TITLE
bug(SELECT): Chrome crashing on select drop down open

### DISCUSSION THREAD

**Discussion by JoeMellon** [angular/components#28911] [General]
### Is this a regression?

- [X] Yes, this behavior used to work in the previous version

### The previous version in which this bug was not present was

14, 16, 17

### Description

On the latest Chrome version 124.0.6367.61 opening mat-select drop down causes the browser to crash.

### Reproduction

StackBlitz link: 
Steps to reproduce:
1. Go to https://v15.material.angular.io/components/select/overview
2. Open drop down 

### Expected Behavior

Chrome not crashing.

### Actual Behavior

Chrome crashing.

### Environment

- Angular: 15
- CDK/Material: 15
- Browser(s): Chrome
- Operating System (Windows):

**Comment by dac-humi** [comment]
Also seen on: https://issues.chromium.org/issues/335553723

**Comment by zarend** [comment]
Hello, Ty for linking to the Chromium issue. Unfortunately, if this is a Chrome issue, then there's nothing that Angular components can do about this.

I don't think this is an issue to Angular components. I'm going to convert this to a discussion so developers can comment on this and share information

-Zach

**Comment by JoeMellon** [comment]
@zarend How can this be closed instantly as a Chrome issue? this is going to impact a huge amount of apps globally and could quite clearly be an Angular issue. If I remove the `mat-label` element the drop down works, if I use a native select within the form field the drop down works, if I inspect the dom and remove the `aria-owns` attribute the drop down works. Is there not going to be any investigation at all from the Angular team on this?

**Reply by IsaacPCooper** [reply]
Agree with Joe above, if this was purely a chrome issue it would be causing issues for other JS frameworks, surely?

If aria-owns is added to a html element chrome doesn't crash so does seem to be specific to angular components. @zarend

**Comment by alx-andru** [comment]
FYI for keeping track of the issue within Angular Material
https://github.com/angular/components/issues/28905
